dev/Dockerfile.base

```diff
@@ -32,6 +32,7 @@ RUN set -ex; \
 RUN set -ex; \
     touch /var/lib/rpm/* \
     && dnf -y install \
+        gettext \
```

Comment:
What changed that makes gettext optional now?

Reply:
It's used when building django translations.